packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -39,7 +39,7 @@ contract L1StandardBridge_Receive_Test is Bridge_Initializer {
         assertEq(address(op).balance, 0);
 
         vm.expectEmit(true, true, true, true);
-        emit ETHBridgeInitiated(alice, alice, 100, hex"");
```

Comment:
Ah, the thing is that _both_ events should be emitted here.

Reply:
I think this test _might_ still pass if you restore this line.

Reply:
good catch!